vtm/src/org/oscim/tiling/source/mapfile/MapDatabase.java

```diff
@@ -1248,60 +1278,5 @@ public float projectLat(double lat) {
         public float projectLon(double lon) {
             return (float) (lon / divx - dx);
         }
-
-        void project(MapElement e) {
```

Comment:
No serious need for now to remove the method.

Reply:
okay. But why?  Function is not used anymore and implementation was causing the problem.
Functionality moved somewhere else so it is obsolete.

Reply:
`TileProjection` is an utility class, which could be even in its own file and its public API could be called from other places as well. So nothing wrong if maintain it in its original form.

Reply:
i see. alright :)